templates/environment/cf.yml

```diff
@@ -227,9 +227,32 @@ Resources:
       LoadBalancerArn: !Ref PublicLoadBalancer
       Port: 80
       Protocol: HTTP
+  
+  CloudformationExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: !Sub ${AWS::StackName}-CFNExecutionRole
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+        - Effect: Allow
+          Principal:
+            Service: 'cloudformation.amazonaws.com'
+          Action: sts:AssumeRole
+      Path: /
+      Policies:
+        - PolicyName: execute
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+            - 
+              Effect: Allow
+              Action: "*"
```

Comment:
Is this copied from somewhere? Just anticipating appsec saying this is too permissive.

/c @allisaurus

Reply:
Please use the PowerUser managed policy and the ability to create roles which look like task execution/task role.

Reply:
(also the permissive policy is going to be mitigated by allowing folks to pass in their own policy, if they want `archer env init --app-infrastructure-policy app-policy.json`)

Reply:
Nope not copied I wrote that myself based on a previous casual discuss I had with David. But 100% agree Daivid's suggestion is better.